src/gpodder/gtkui/main.py

```diff
@@ -210,7 +209,15 @@ def changed_cb(spinbutton):
         # Set up the first instance of MygPoClient
         self.mygpo_client = my.MygPoClient(self.config)
 
-        self.inject_extensions_menu()
+        # Extensions section in app menu and menubar Extras menu
+        extensions_menu = Gio.Menu()
+        self.application.app_menu.insert_section(2, "Extensions", extensions_menu)
```

Comment:
This places the extensions in the gPodder **and** Extras menus.

Reply:
Oops, I've now removed the extensions from the app menu.